CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Fix visibility of permissions and canonicalize upper/lower case. Fixes UICR-122 and UICR-123.
 * Added ability to view and edit a course's `numberOfStudents` field. Fixes UICR-128
+* Replaced current permission sets with ones that can be i18n'ed. Fixes UICR-130.
```

Comment:
Fair point. I've always thought of the atomic permissions that the backend defines as permissions and every bag o' perms as permission sets and so have mulishly stuck to a permission _set_ nomenclature because it encapsulates the fact that it consists of multiple permissions. But yeah, I think your wording is cleaner. The rest of my life starts today!